core/runtime/src/syscalls_impl_riscv.rs

```diff
@@ -0,0 +1,82 @@
+use libtock_platform::{OneArgMemop, RawSyscalls, YieldType, ZeroArgMemop};
+
+impl RawSyscalls for crate::TockSyscalls {
+    // This yield implementation is currently limited RISC-V versions without
+    // floating-point registers, as it does not mark them clobbered.
+    #[cfg(not(any(target_feature = "d", target_feature = "f")))]
+    fn raw_yield(r0_in: YieldType) -> u32 {
+        let mut r0 = r0_in as u32;
+        let mut _a4 = 0;
```

Comment:
This shouldn't have an underscore right?

Reply:
I agree, but removing the underscore gives an "unused variable" warning. I could do the following instead:

```
#[allow(unused)]  // False positive lint
let mut a4 = 0;
```

I could also add a comment like `// underscore to avoid false positive unused variable warning` to this.

Reply:
That's annoying. I would say that `#[allow(unused)]  // False positive lint` is probably better, but your call.

Reply:
Turns out I was holding it wrong, and `inlateout("x14") 0 => _` is the proper way to express this. Fixed in latest commit.